test-cases/tests/walkthroughs/g16-verify-settings-notifications-pop-up.md

```diff
@@ -0,0 +1,19 @@
+---
+estimate: 15m
+tags:
+  - 2.6.0
+---
+
+# G16 - Verify settings notifications pop up
+
+Acceptance Criteria:
+
+When logged into solution explorer as a customer-admin
```

Comment:
not `customer-admin`, but `dedicated-admin` ? Even more precise would be to say `a user from didicated-admin group` but that's probably an overkill

Reply:
Agree, updated with commit